contracts/interchain-token-service/src/msg.rs

```diff
@@ -9,6 +9,13 @@ use router_api::{Address, ChainNameRaw};
 use crate::state::{TokenConfig, TokenInstance};
 use crate::TokenId;
 
+// Pagination limit
+const DEFAULT_LIMIT: u32 = 30;
```

Comment:
just call the parameter DEFAULT_PAGINATION_LIMIT, then the comment isn't necessary